src/super_gradients/training/models/detection_models/yolo_base.py

```diff
@@ -691,7 +696,7 @@ def replace_head(self, new_num_classes=None, new_head=None):
 
             new_last_layer = DetectX(
                 num_classes=new_num_classes,
-                stride=self._head.anchors.stride,
+                stride=self.strides,
```

Comment:
Not sure if I understand, this single line was the bug right? It was just forgotten in the previous PR right?

Reply:
Exactly. Removed the self._head.anchors.stride, but did forget to test replace_head still works